AppCenter/AppCenter/Internals/Util/MSConstants+Internal.h

```diff
@@ -90,3 +91,22 @@ static const NSUInteger kMSPersistenceFlagsMask = 0xFF;
  * Common schema prefix separator used in various field values.
  */
 static NSString *const kMSCommonSchemaPrefixSeparator = @":";
+
+// Encryption constants.
+static int const kMSEncryptionAlgorithm = kCCAlgorithmAES;
```

Comment:
Are these constants intended to be used by the encrypter only? If it's not useful to other classes then they can be moved to the encrypter private header.

Reply:
Fixed.